src/pages/Dashboard/UserAccountCard/UserAccountCard.tsx

```diff
@@ -12,8 +12,7 @@ import UserAccountCardLoader from './UserAccountLoader';
 
 
 const UserAccountCard: FC = () => {
-
-    const actualTheme = useAppSelector(state => state.persistedThemeSlice.theme);
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)